Rocket.Chat.SDKTests/Managers/LivechatManagerTests.swift

```diff
@@ -12,6 +12,7 @@ import SwiftyJSON
 
 class LivechatManagerTests: XCTestCase {
 
+    // swiftlint:disable:next force_unwrapping
     let socket = WebSocketMock(url: URL(string: "http://doesnt.matter")!)
```

Comment:
You disabled it in many places to the same code... maybe wrap the URL into a method that will return a valid URL? I think it would be much better.